src/components/views/Tables/Tables.js

```diff
@@ -1,14 +1,165 @@
+  },
+  {
+    hour: '14:00 - 14:29',
+    tables: [
+      { table: 1, status: 'booked', bookingId: 45356 },
+      { table: 2, status: 'free', bookingId: null },
+      { table: 3, status: 'free', bookingId: null },
+      { table: 4, status: 'free', bookingId: null },
+    ],
+  },
+  {
+    hour: '14:30 - 14:59',
+    tables: [
+      { table: 1, status: 'booked', bookingId: 8764 },
+      { table: 2, status: 'event', bookingId: 87456 },
+      { table: 3, status: 'booked', bookingId: 5321 },
+      { table: 4, status: 'free', bookingId: null },
+    ],
+  },
+  {
+    hour: '15:00 - 15:29',
+    tables: [
+      { table: 1, status: 'event', bookingId: 5454 },
+      { table: 2, status: 'event', bookingId: 2521 },
+      { table: 3, status: 'event', bookingId: 5643 },
+      { table: 4, status: 'free', bookingId: null },
+    ],
+  },
+  {
+    hour: '15:30 - 15:59',
+    tables: [
+      { table: 1, status: 'booked', bookingId: 8764 },
+      { table: 2, status: 'booked', bookingId: 8746 },
+      { table: 3, status: 'free', bookingId: null },
+      { table: 4, status: 'free', bookingId: null },
+    ],
+  },
+  {
+    hour: '16:00 - 16:29',
+    tables: [
+      { table: 1, status: 'booked', bookingId: 3490 },
+      { table: 2, status: 'event', bookingId: 1295 },
+      { table: 3, status: 'booked', bookingId: 3486 },
+      { table: 4, status: 'booked', bookingId: 1123 },
+    ],
+  },
+  {
+    hour: '16:30 - 16:59',
+    tables: [
+      { table: 1, status: 'booked', bookingId: 8764 },
+      { table: 2, status: 'event', bookingId: 87456 },
+      { table: 3, status: 'booked', bookingId: 5321 },
+      { table: 4, status: 'free', bookingId: null },
+    ],
+  },
+];
+
+const renderActions = (status, table) => {
+  switch (status) {
+    case 'free':
+      return (
+        <Button to={`${process.env.PUBLIC_URL}/tables/booking/new`} component={NavLink}>Free</Button>
+      );
+    case 'booked':
+      return (
+        <Button to={`${process.env.PUBLIC_URL}/tables/booking/${table.bookingId}`} component={NavLink}>{table.status}</Button>
+      );
+    case 'event':
+      return (
+        <Button to={`${process.env.PUBLIC_URL}/tables/events/${table.bookingId}`} component={NavLink}>{table.status}</Button>
+      );
+    default:
+      return null;
+  }
+};
 
 const Tables = () => (
-  <div className={styles.component}>
-    <h2>Tables</h2>
-    <Link to={`${process.env.PUBLIC_URL}/tables/booking/new`} activeClassName='active'>New Booking</Link>
-    <Link to={`${process.env.PUBLIC_URL}/tables/booking/:id`} activeClassName='active'>Booking ID</Link>
-    <Link to={`${process.env.PUBLIC_URL}/tables/events/new`} activeClassName='active'>New Event</Link>
-    <Link to={`${process.env.PUBLIC_URL}/tables/events/:id`} activeClassName='active'>Event ID</Link>
+  <div>
+    <div className={styles.component}>
+      <h2>Tables</h2>
+      <Link to={`${process.env.PUBLIC_URL}/tables/booking/new`} activeClassName='active'>New Booking</Link>
```

Comment:
Stwórz nowy plik `config.js` i wyeksportuj tam stałą `PUBLIC_URL`, której wartością będzie `process.env.PUBLIC_URL`. Następnie wszędzie gdzie używasz `process.env.PUBLIC_URL` zaimportuj stałą `PUBLIC_URL` z pliku config.js i podmień jej wykorzystanie. Np. tutaj na `<Link to={'${PUBLIC_URL}/tables/booking/new'} activeClassName='active'>New Booking</Link>`. Postaraj się wykorzystać w Twoim edytorze funkcje podmiany wielu miejsc jednocześnie (`Znajdź i zamień`), aby ograniczyć ręczne modyfikacje do minimum